Modelica_LinearSystems2/package.mo

```diff
@@ -11,12 +11,12 @@ package Modelica_LinearSystems2 "Modelica_LinearSystems2 (version 3.0.0) - Analy
 annotation (
   preferredView="info",
   uses(
-    Complex(version="4.0.0"),
-    Modelica(version="4.0.0"),
-    DymolaCommands(version="1.17")),
-  version="3.0.0",
-  versionDate="2024-06-26",
-  dateModified = "2024-06-26 14:00:00Z",
+    Complex(version="4.1.0"),
+    Modelica(version="4.1.0"),
+    DymolaCommands(version="1.20")),
```

Comment:
@DagBruck  I can only see  DymolaCommands **v1.19** in (current) Dymola 2025x Refresh 1, 2025-05-16.